packages/create-itp-react-app/lib/installNPMPackages.js

```diff
@@ -1,4 +1,4 @@
-const shell = require('shelljs');
+const shell = require ('shelljs');
```

Comment:
prob unwanted?

Reply:
don't like it either, but default by prettier says @easybird